src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/EntityCreateController.php

```diff
@@ -63,6 +63,7 @@ public function __construct(
     #[Route(
         path: '/entity/create/type/{serviceId}/{targetEnvironment}/{inputId}',
         name: 'entity_type',
+        requirements: ['inputId' => '.+'],
```

Comment:
Perhaps link to the why (in comment)? Either link to the story here, or maybe to the SF docs?

`https://symfony.com/doc/6.4/routing.html#slash-characters-in-route-parameters`

Reply:
I think it is in the commit message